### PR TITLE
ECAM clear and recall buttons

### PR DESCRIPTION
**Pull request by tyler58546**
**Summary of Changes**

*  Added functional CLR and RCL buttons 
*  Master caution/warning will now automatically deactivate when messages are removed from ecam

The light on the clear button does not illuminate because it is missing a SEQ2 emission texture.

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 1]
```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
...
                         output[i].messages.push(message);
                     }
                 }
             }
             return output;
         }
 
+        clearHighestCategory() {
+            const activeFailures = this.getActiveFailures();
+            for (const n in activeFailures) {
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1522]: Avoid single-letter variables.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 2]
```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
...
+                var category = activeFailures[n];
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1523]: Can be `const`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 3]
```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
         getActiveFailures() {
             var output = {};
             this.hasActiveFailures = false;
+            this.hasWarnings = false;
+            this.hasCautions = false;
             for (var i = 0; i < this.messages.failures.length; i++) {
                 const messages = this.messages.failures[i].messages;
                 for (var n = 0; n < messages.length; n++) {
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1501]: Should be `let`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 4]
```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
         getActiveFailures() {
             var output = {};
             this.hasActiveFailures = false;
+            this.hasWarnings = false;
+            this.hasCautions = false;
             for (var i = 0; i < this.messages.failures.length; i++) {
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1499]: Should be `let`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 5]
```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
         getActiveFailures() {
             var output = {};
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1495]: Should be `let`.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js (hunk 3 above)
Comment (tyler58546) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1501]: This is used outside the method.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js (hunk 4 above)
Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1499]: @Benjozork tyler didn't touch that....

Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1499]: actually, maybe i cant tell what line you're talking about

**Pull request merged**